source/CodeMirror.mint

```diff
@@ -1,22 +1,22 @@
 /* A component that integrates the CodeMirror editor. */
 component CodeMirror {
+  connect CodeMirror.Store exposing { addEditor }
+
   /* The JavaScript files of Codemirror to load, either locally or from a CDN. */
   property javascripts : Array(String) = [
-    "https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0" \
-    "/codemirror.min.js"
+    "https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/codemirror.min.js"
   ]
 
   /* The CSS files of Codemirror to load, either locally or from a CDN. */
   property styles : Array(String) = [
-    "https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0" \
-    "/codemirror.min.css"
+    "https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/codemirror.min.css"
   ]
 
   /* Handler for the change event. */
-  property onChange : Function(String, a) = ((value : String) : Void => { void })
+  property onChange : Function(String, Promise(Never, Void)) = ((value : String) : Promise(Never, Void) => { sequence { Promise.never() }})
```

Comment:
It's better to use `Function(String, a)` because it  does not limit the return value.